plugins/@grouparoo/demo/src/connections/mysql.ts

```diff
@@ -0,0 +1,201 @@
+import mysql from "mysql";
+import { log } from "actionhero";
+import Connection from "../util/connection";
+import { promisify } from "util";
+
+const TYPES = {
+  users: {
+    id: "INT NOT NULL PRIMARY KEY",
+    account_id: "INT",
+    first_name: "VARCHAR(191) NOT NULL",
+    last_name: "VARCHAR(191) NOT NULL",
+    email: "VARCHAR(191) NOT NULL",
+    gender: "VARCHAR(191)",
+    ip_address: "VARCHAR(191)",
+    avatar: "VARCHAR(191)",
+    language: "VARCHAR(191)",
+    deactivated: "BOOLEAN",
+    created_at: "TIMESTAMP NOT NULL",
+    updated_at: "TIMESTAMP NOT NULL",
+  },
+
+  purchases: {
+    id: "INT NOT NULL PRIMARY KEY",
+    user_id: "INT NOT NULL",
+    item: "VARCHAR(191) NOT NULL",
+    category: "VARCHAR(191) NOT NULL",
+    price: "DECIMAL",
+    state: "VARCHAR(191)",
+    created_at: "TIMESTAMP NOT NULL",
+  },
+
+  accounts: {
+    id: "INT NOT NULL PRIMARY KEY",
+    name: "VARCHAR(191) NOT NULL",
+    domain: "VARCHAR(191) NOT NULL",
+    plan_id: "INT",
+    created_at: "TIMESTAMP NOT NULL",
+    updated_at: "TIMESTAMP NOT NULL",
+  },
+
+  plans: {
+    id: "INT NOT NULL PRIMARY KEY",
+    name: "VARCHAR(191) NOT NULL",
+    seats: "INT NOT NULL",
+    monthly_rate: "INT NOT NULL",
+  },
+
+  payments: {
+    id: "INT NOT NULL PRIMARY KEY",
+    account_id: "INT NOT NULL",
+    amount: "DECIMAL",
+    state: "VARCHAR(191)",
+    created_at: "TIMESTAMP NOT NULL",
+  },
+};
+
+function findConfig(): any {
```

Comment:
Are these `any`'s because they're generic?

Reply:
I'm sure there's a better type, even just key/value, but I didn't prioritize figuring it out.